pkg/util/steps/condition.go

```diff
@@ -24,15 +24,17 @@ type conditionFunction func(context.Context) (bool, error)
 // The Condition will execute f repeatedly (every Runner.pollInterval), timing
 // out with a failure when more time than the provided timeout has elapsed
 // without f returning (true, nil). Errors from `f` are returned directly.
-func Condition(f conditionFunction, timeout time.Duration) conditionStep {
+func Condition(f conditionFunction, timeout time.Duration, fail bool) conditionStep {
```

Comment:
The behavior of `Condition` changed with addition of `fail` parameter, which can be seen to have different meanings. How easy would the function be with updated documentation header with the right meaning?